contracts/misc/Looping.sol

```diff
@@ -56,8 +58,8 @@ contract Looping {
     require(amount > 0, "Need to attach Ether");
     WETH.deposit{value: amount}();
 
-    (uint256 deposited, uint256 borrowed) = internalLoop(user, address(WETH), amount, borrowRatio, loopCount);
-    emit LoopETH(user, amount, deposited, borrowed, borrowRatio, loopCount);
+    InternalLoopVariables memory variables = internalLoop(user, address(WETH), amount, borrowRatio, loopCount);
```

Comment:
shall we check loopCount > 0 as per the audit suggestion?

Reply:
loopCount = 0 is used when leverage is 1x, we didn't face the issue that audit mentioned becasue `deposit` will be called one time when loopCount = 0

Reply:
sounds good